syntax = "proto3";

package message;

service MessageService {
    rpc ProcessText(BinaryRequest) returns (TextResponse);
}

message BinaryRequest {
    bytes pdf_bytes = 2;
}

message TextResponse {
    string processed_content = 1;
}
